grant/default.go

```diff
@@ -18,7 +18,8 @@ func GetDefaultGrant(server *state.Server, globalCollectionOpts state.Collection
 
 	req.Header.Set("Pganalyze-Api-Key", server.Config.APIKey)
 	req.Header.Set("Pganalyze-System-Id", server.Config.SystemID)
-	req.Header.Set("Pganalyze-System-Type", server.Config.SystemType)
+	// TODO(ianstanton) Placeholder for local dev
+	req.Header.Set("Pganalyze-System-Type", "self_hosted")
```

Comment:
We need to make a server side change to support SystemType `tembo`. We'll revert this once that change is made.